lib/babel-schmooze-sprockets/engine.rb

```diff
@@ -1,7 +1,4 @@
 module BabelSchmoozeSprockets
   class Railtie < Rails::Engine
-    initializer "babel-schmooze-sprockets" do
-      config.assets.paths << File.expand_path("#{__dir__}/../../node_modules")
-    end
   end
 end
```

Comment:
as far as I remember it adds the `app/assets/javascripts` directory into the assets load path for us.